content/Solutions/Integrations/Containers.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: "Containers"
-description: "Guide to configure TrueNAS CORE as backing storage for container solutions."
+description: "Guide to configure TrueNAS as backing storage for container solutions."
 weight: 25
 aliases:
   - /core/solutions/integrations/containers/
 ---
 
-TrueNAS CORE & Enterprise can both be used as backing storage for container workloads.
+TrueNAS & TrueNAS Enterprise can both be used as backing storage for container workloads.
```

Comment:
can't say "both" because we don't mention two options in this sentence, so how about either of these:
```suggestion
TrueNAS & TrueNAS Enterprise can be used as backing storage for container workloads.
```
or if you want to expand this
```suggestion
Linux-based TrueNAS and FreeBSD-based TrueNAS Enterprise can both be used as backing storage for container workloads.
```